src/main/scala/gemmini/LoopMatmul.scala

```diff
@@ -22,6 +22,7 @@ class LoopMatmulLdAReq(val block_size: Int, val coreMaxAddrBits: Int, val iterat
   val transpose = Bool()
   val addr_start = UInt(log2Up(max_addr).W)
   val loop_id = UInt(log2Up(concurrent_loops).W)
+  val is_resadd = Bool()
```

Comment:
Does this mean that the LoopMatmul FSM will be able to do _either_ resadds _or_ matmuls, but not both at the same time? I think we could get higher utilization if we overlap the resadds with matmuls/convs, since resadds have low arithmetic intensity while convs/matmuls are oftentimes compute-bound

Reply:
yup, it would be either resadd or matmul. The current Gemmini has separate resadd/matmul function anyways, so I didn't add that feature. 
and sure, we can add that feature, but it would need more work on top of this.